tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -917,6 +981,15 @@ public:
 
    void Finalize();
 
+   // Helper functions for RMergeableValue
+   void *GetMergeableValue()
+   {
+      const ULong64_t counts = std::accumulate(fCounts.begin(), fCounts.end(), 0);
```

Comment:
this silently casts counts to int (classic std::accumulate pitfall), please use 0ull as init value